cloudformation/content-authorisation-proxy.cf.yaml

```diff
@@ -24,6 +24,9 @@ Parameters:
     AllowedValues:
     - CODE
     - PROD
+  App:
+    Type: String
+    Default: content-authorisation-proxy
```

Comment:
why did you add App as a parameter?

Reply:
I prefer using a parameter for this, otherwise we have to write "content-authorisation-proxy" multiple times

Reply:
ok. but doesn't that mean that the app name can be changed when updating the stack ?

Reply:
Yes, theoretically they could have done. This should be prevented by https://github.com/guardian/content-authorisation-proxy/pull/83/commits/47e2a719bb22d2b891158e9780c0c604ed2f38ed